content/en/docs/home/contribute/localization.md

```diff
@@ -8,47 +8,87 @@ approvers:
 
 {{% capture overview %}}
 
-We're happy to add localizations (l10n) of Kubernetes documentation to the website!
+The Kubernetes documentation is currently available in [multiple languages](#supported-languages) and we encourage you to add new localizations ([l10n](https://blog.mozilla.org/l10n/2011/12/14/i18n-vs-l10n-whats-the-diff/))!
 
-Localizations must meet the following requirements for _workflow_ (how to localize) and _output_ (what to localize).
+In order for localizations to be accepted, however, they must fulfill some requirements related to [workflow](#workflow) (*how* to localize) and [output](#output) (*what* to localize).
 
 {{% /capture %}}
 
 {{< toc >}}
 
 {{% capture body %}}
 
-## Workflow  
+## Supported languages
+
+The Kubernetes documentation is currently available in the following languages:
+
+{{< languages-list >}}
 
-All l10n work must be stored and tracked within the [Kubernetes organization](https://github.com/kubernetes).
+## Workflow  
 
-### Basis for localizations
+All Kubernetes documentation for all languages is built from the [kubernetes/website](https://github.com/kubernetes/website) repository on GitHub. You don't need to create or maintain a separate repository to add a localization.
```

Comment:
AFAICT, this is not the current agreement. We have some translations forked from the English site and volunteers to focus on creating and reviewing translations. The outputs, however, are supposed to be synced back the the kubernetes/website repo. This is to reduce noises on the "main" site and to have l10n contributors more focused on their jobs.